### PR TITLE
fix: bump contract and fix capabilities

### PR DESCRIPTION
**Pull request by SebastianElvis**
Before merging:

- [x] Wait until https://github.com/babylonchain/babylon-contract/pull/183 is merged and tagged

### CODE REVIEW

**Comment by maurolacy** [comment]
Here: https://github.com/babylonchain/babylon-contract/releases/tag/v0.7.0-rc.2

File: demo/app/wasm.go [hunk 1]
```diff
@@ -11,6 +11,8 @@ func AllCapabilities() []string {
 		"cosmwasm_1_1",
 		"cosmwasm_1_2",
 		"cosmwasm_1_3",
+		"cosmwasm_1_4",
+		"cosmwasm_2_0",
```
Comment (maurolacy) [inline demo/app/wasm.go:15]: It seems `cowsmwasm_2_0` implies all of the preceding ones. See the https://github.com/CosmWasm/cosmwasm/blob/main/MIGRATING.md#15x---20x bullets.

File: tests/testdata/version.txt [hunk 2]
```diff
@@ -1 +1 @@
-v0.7.0-rc.0
+8d8734fd21795d160149bcfff759172f4a6874f9
```
Comment (maurolacy) [inline tests/testdata/version.txt]: `v0.7.0-rc.2` just out of the oven.

**Review by maurolacy — APPROVED** [review]

File: demo/app/wasm.go (hunk 1 above)
Comment (SebastianElvis) [reply demo/app/wasm.go:15]: Hmm it seems that we still need all of these -- it's the CW contract that the `cosmwasm_2_0` feature implies the prior versions. If the [chain](https://github.com/CosmWasm/cosmwasm/blob/main/docs/CAPABILITIES-BUILT-IN.md) wants to support CW contract with lower CW versions we still need those capabilities. Keeping them for now

**Pull request merged**